tests/events/Anchor.toml

```diff
@@ -1,3 +1,6 @@
+[features]
+seeds = true
```

Comment:
I think instead of using opt-in feature `seeds`, we can supply these accounts when needed from client side.